src/Microsoft.AspNetCore.ResponseCaching/Interfaces/IResponseCacheKeyProvider.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-
-namespace Microsoft.AspNetCore.ResponseCaching
-{
-    public interface IResponseCacheKeyProvider
```

Comment:
I don't think the interface is valuable to keep? We only have one internal implementation of it anyways.